package.json

```diff
@@ -7,7 +7,7 @@
   "author": "Holograph",
   "scripts": {
     "build": "turbo run build",
-    "dev": "turbo run dev",
+    "dev": "dotenv turbo run dev",
```

Comment:
curious what this does differently?

Reply:
supposedly uses a .env file that was configured in turbo.json

Reply:
cool